app/models/spree/user_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Spree::User.class_eval do
+  def full_name
```

Comment:
@jhawthorn I have pushed the change I proposed about a concern in the decorators. That would allow us to remove this `full_name` method. Let me know which route is better, or if you see a different route.